pkgs/development/libraries/libint/default.nix

```diff
@@ -139,7 +139,7 @@ let
         tests/eri/Makefile \
         tests/hartree-fock/Makefile \
         tests/unit/Makefile; do
-          substituteInPlace $f --replace "/bin/rm" "rm"
+          substituteInPlace $f --replace-warn "/bin/rm" "rm"
```

Comment:
I mean if `/bin/rm` is not matched then there is not really a problem. Either there is `rm`, which is fine, or there is no rm at all, which is also fine :)

Reply:
if there is no `/bin/rm` maybe it should fail and file path  needs to be remove from `for` loop ?

Reply:
As far as I remember, the purpose (and default behavior) of `substituteInPlace` is to fail if the pattern is not found. For that reason it it preferred over, for example  `sed`, so that the code can be removed when it it is not needed anymore.

Reply:
Yes but substituteInPlace is applied to multiple files here and matches multiple of them: https://github.com/search?q=repo%3Aevaleev%2Flibint%20%2Fbin%2Frm&type=code

A warning is fine for me, it will still be logged if some file misses the match and otherwise this is just a harmless normalisation. The introduction of `replace-warn` would be wild if it would be intended not to be used.

Reply:
> A warning is fine for me, it will still be logged if some file misses the match and otherwise this is just a harmless normalisation. The introduction of `replace-warn` would be wild if it would be intended not to be used.

There are always edge cases where a warning is better than a fail. I guess this is one of the edge cases where a warning instead of a fail is better.